hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegion.java

```diff
@@ -5066,6 +5068,26 @@ private static void updateCellTimestamps(final Iterable<List<Cell>> cellItr, fin
     }
   }
 
+  /**
+   *
+   * @param splitRow
+   * @param regionStartKey
+   * @return Whether the split row is valid relative to the region's start key
+   */
+  private static boolean isValidSplit(byte[] splitRow, byte[] regionStartKey) {
```

Comment:
I didn't see a need to specify a prefix length of 1, I think the following invariants should hold true for any prefix length